Makefile

```diff
@@ -34,6 +34,7 @@ all:
 
 bootstrap:
 	${DEV_ENV_CMD} glide install
+	go get github.com/pwaller/goupx
```

Comment:
See  deis/docker-go-dev#11. We'll probably have to bag and tag another revision of `deis/go-dev` once again, but once that's done we can use that.